amp/WEB-INF/src/org/digijava/module/aim/action/AuditLoggerManager.java

```diff
@@ -61,11 +65,30 @@ public ActionForward modePrepare(ActionMapping mapping, ActionForm form,
             }
                 
         }
+
         Collection<AmpAuditLogger> logs=AuditLoggerUtil.getLogObjects(vForm.isWithLogin());
```

Comment:
if filtering is done we don't need to call getLogObjects since you will be having an additional query that is not needed, you will be overriding logs collection

Reply:
so we need to move the filtering to another method? since we need to call getLogObjects at page load

Reply:
Nope, I mean that if you are filtering, then you need to only call getLogObjects, if you are not filtering (page loads) you need to call this method that get all objects. you should, in the same action, call one or the other